src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -63,8 +54,6 @@ public void testDefaultTimestamp() throws InvalidParametersException, EntityAlre
         ______TS("success : defaultTimeStamp for updatedAt date");
         
         assertEquals(defaultStudentCreationTimeStamp, student.getUpdatedAt());
-        
-        studentsDb.deleteStudent(s.course, s.email);
```

Comment:
Not entirely sure, but why is this deleted? Isn't this PR about adding more of this stuff, or at least retaining the existing ones?